src/plugins/openmeteo/datasource.py

```
@@ -0,0 +1,333 @@
+                        yield measurements
+                        measurements = []
+                        
+                if measurements:
+                    yield measurements
+                    
+                if limit and measurement_count >= limit:
+                    break
+                    
+                current_start = chunk_end + timedelta(days=1)
+                
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)